config.json

```diff
@@ -21,93 +21,59 @@
   ],
   "methods": [
     {
-      "name": "executeTerraformInit",
-      "viewName": "Init",
+      "name": "getTerraformVersion",
+      "viewName": "Get Terraform Version",
+      "params": []
+    },
+    {
+      "name": "runMainCommand",
+      "viewName": "Run Terraform Main Command",
       "params": [
         {
-          "name": "workingDirectory",
-          "viewName": "Working Directory",
-          "type": "string",
+          "name": "mode",
+          "viewName": "Mode",
           "required": true,
-          "description": "This is the location where Terraform will expect to find an inventory and execution script.",
-          "placeholder": "/home/user/tmp/",
-          "learnUrl": "https://www.terraform.io/docs/cli/commands/init.html"
+          "type": "options",
+          "options": [
+            {
+              "id": "init",
+              "name": "Init"
+            },
+            {
+              "id": "plan",
+              "name": "Plan"
+            },
+            {
+              "id": "apply",
+              "name": "Apply"
+            },
+            {
+              "id": "destroy",
+              "name": "Destroy"
```

Comment:
Easy enough to add "Validate" as well, since are here? Validate is like Init - except it doesn't even take the upgrade parameter. No parameters at all for Validate - only working directory.